config/jobs/build-drivers/build-new-bottlerocket.yaml

```diff
@@ -6,14 +6,13 @@ presubmits:
     agent: kubernetes
     branches:
       - ^master$
-    run_if_changed: '^driverkit/config/[a-z0-9.+-]{5,}/(.+/)?bottlerocket_4*'
+    run_if_changed: '^driverkit/config/[a-z0-9.+-]{5,}/(.+/)?bottlerocket*'
```

Comment:
Avoid having bottlerocket_4 and bottlerocket_5 jobs; Unify them, since there are not so many.